howtos/v3_to_v4.rst

```diff
@@ -170,19 +176,19 @@ Many bug fixes
 ^^^^^^^^^^^^^^
 
 In the process of moving from a CentOS base to a "Built on openSUSE" one,
-we have found and fixed a large number of bug, and inherited such things as our
+we have found and fixed a large number of bugs, and inherited such things as our
 `Rockstor 4 Installer Recipe <https://github.com/rockstor/rockstor-installer>`_
 that trivially enables highly customised installer creation.
 We also now have ARM64 (e.g. Pi4 / Ten64) compatibility, baring some Rock-ons,
 courtesy of openSUSE's extreme heritage in ARM support.
 
 Also note the following, now we are past the `Jump <https://en.opensuse.org/Portal:Jump>`_ initiative:
 
-- In v3 our upstream of CentOS had in turn it's upstream of RedHat's RHEL.
-- In v4 our upstream of openSUSE has in turn it's increasing binary compatible with SuSE SLES.
+- In v3 our upstream of CentOS had in turn its upstream of RedHat's RHEL.
+- In v4 our upstream of openSUSE has in turn its increasingly binary compatible upstream with SuSE SLES.
```

Comment:
```suggestion
- In v4 our upstream of openSUSE has in turn its binary compatible upstream of SuSE SLES.
```

Reply:
Now we are past a Leap 15.3 (Jump) and onto 15.4 I think we can say this: given the Jump initiative is behind us and. With 15.4 being current and our preferred download. Plus they now share repositories as a result.

Reply:
yep, that also makes perfect sense